### ISSUE TITLE
Getaway Black Monday (Graphical Issue)

### ISSUE THREAD

**Issue by mrchrismania** [PCSX2/pcsx2#1846]
pcsx2-v1.5.0-dev-1911-g0a5f796-windows-x86

Emulation Settings used:
Safe faster Preset + SuperVU recompiler to fix enemies being able to shoot through walls

im getting weird green flashes come up on the screen sometimes during cutscenes and during gameplay using the OpenGL renderer also sometimes when i die it comes up with this
![untitled](https://cloud.githubusercontent.com/assets/20265191/23601763/0862ccb6-0245-11e7-865c-88b89f7260a9.png)

PC Specs:
 Skylake i7 6700 4.0ghz
MSI Armor GTX 1070
16GB DDR4 2133mhz Dual Channel
280GB SSD + 1TB HDD
Windows 10 Home 64 Bit

**Comment by MrCK1** [comment]
What happens when you use the DX11 renderer?

**Comment by gregory38** [comment]
DX will skip the bad draw call. You can see the depth buffer in the first screenshot. Potentially it is due to superVU that does a strange rendering and the texture cache fail to detect it. A gsdump of the first image will be nice.

**Comment by mrchrismania** [comment]
MrCK1 it works fine in both software mode and direct3D11 i did notice on direct3D11 the first cutscene in the game after i press new game on the main menu. the cutscene is very blurry using Direct3D11 then afterwards all the cutscenes in the game are completly fine and all work with SuperVU recompiler.

**Comment by mrchrismania** [comment]
yep i will make a gs dump for you when i have the time. @gregory38

**Comment by mrchrismania** [comment]
also sorry for the bad screenshots im using an ultra wide monitor and i dont think it shows the correct resolution when i press print screen.

**Comment by gregory38** [comment]
```
it works fine in both software mode and direct3D11
```
D3D doesn't render any depth effect, instead it just skill the draw calls. So you don't see any bad thing on the screen. But it misses some rendering effects. So you can't call it fine.

**Comment by FlatOutPS2** [comment]
@gregory38 

This game also has a different issue with depth emulation. It makes shadows appear as big squares.

GS dump:
[The Getaway Black Monday - Depth emulation makes (car) shadows appear as big squares.zip](https://github.com/PCSX2/pcsx2/files/853468/The.Getaway.Black.Monday.-.Depth.emulation.makes.car.shadows.appear.as.big.squares.zip)

**Comment by refractionpcsx2** [comment]
Picture corruption seems to have gone, the shadow (due to depth) is being covered by issue https://github.com/PCSX2/pcsx2/issues/3986
